packages/calypso-e2e/tsconfig.json

```diff
@@ -1,10 +1,9 @@
 {
 	"extends": "@automattic/calypso-build/typescript/ts-package.json",
 	"compilerOptions": {
-		"module": "commonjs",
-		"outDir": "dist/cjs",
+		"outDir": "dist/esm",
 		"declarationDir": "dist/types",
 		"rootDir": "."
```

Comment:
Unrelated, but this line can be eliminated as "." is the default.

Reply:
Updated!